sentence_transformers/sparse_encoder/losses/SpladeLoss.py

```diff
@@ -0,0 +1,73 @@
+from __future__ import annotations
+
+from collections.abc import Iterable
+from enum import Enum
+
+import torch
+import torch.nn as nn
+
+from sentence_transformers.sparse_encoder.losses import (
+    FlopsLoss,
+    SparseDistillKLDivLoss,
+    SparseMarginMSELoss,
+    SparseMultipleNegativesRankingLoss,
+)
+from sentence_transformers.sparse_encoder.SparseEncoder import SparseEncoder
+
+
+class PrincipalLoss(Enum):
+    """The principal loss types for the model"""
+
+    MMSE = SparseMarginMSELoss
+    KL = SparseDistillKLDivLoss
+    MRL = SparseMultipleNegativesRankingLoss
+
+
+class SpladeLoss(nn.Module):
+    def __init__(
+        self, model: SparseEncoder, main_loss: PrincipalLoss, lamda_corpus: float = 0.1, lamda_query: float = 0.1
```

Comment:
```suggestion
        self, model: SparseEncoder, main_loss: PrincipalLoss, lambda_corpus: float = 0.1, lambda_query: float = 0.1
```
Probably worth doing a full replace to fix this typo (I reckon it's a typo at least)
![image](https://github.com/user-attachments/assets/41aa9656-fd8a-4dbe-8fb4-47538c90ecf0)